email.rst

```diff
@@ -51,6 +51,7 @@ The Swift Mailer library works by creating, configuring and then sending
 of the message and is accessible via the ``Swift_Mailer`` service. Overall,
 sending an email is pretty straightforward::
 
+// Put PHP Doc here
```

Comment:
I do not understand what you mean with this. What kind of PHPdoc would you like to see here?

Reply:
@xabbuh 
PHPDoc related to these parameters: $name, \Swift_Mailer $mailer but I understand this information is clearly irrelevant to this context.
This pull request was made when I am lost how to send an email but finally I could do it. 
I am going to close it.